doc/tutorials/using_xenomai_as_uos.rst

```diff
@@ -0,0 +1,81 @@
+.. _using_xenomai_as_uos:
+
+Using Xenomai as User OS
+########################
+
+Build Xenomai kernel
+********************
+
+#. Clone Xenomai kernel souce code::
+
+	# git clone -b F/4.19.59/base/ipipe/xenomai_3.1 https://github.com/intel/linux-stable-xenomai
+
+#. Enter the directory and select default ACRN configuration::
+
+	# cd linux-stable-xenomai && make acrn_defconfig
+
+#. 	Then just start building the kernel::
+
+	# make targz-pkg
+
+#. Upon building completion, you will see a tarball containing kernel and its modules.
+
+   .. code-block:: none
+      :emphasize-lines: 22
+
+      # ls -l
+      total 97944
+      drwxr-xr-x  27 tw tw     4096 Feb 20 10:23 arch
+      drwxr-xr-x   3 tw tw    12288 Feb 21 11:01 block
+      -rw-r--r--   1 tw tw   789264 Feb 25 16:17 built-in.a
+      drwxr-xr-x   2 tw tw     4096 Feb 21 11:01 certs
+      -rw-r--r--   1 tw tw      423 Feb 20 10:17 COPYING
+      -rw-r--r--   1 tw tw    98741 Feb 20 10:17 CREDITS
+      drwxr-xr-x   4 tw tw    12288 Feb 21 11:01 crypto
+      drwxr-xr-x 120 tw tw    12288 Feb 20 10:17 Documentation
+      drwxr-xr-x 143 tw tw     4096 Feb 21 10:48 drivers
+      drwxr-xr-x   2 tw tw     4096 Feb 20 10:21 firmware
+      drwxr-xr-x  73 tw tw    12288 Feb 21 11:01 fs
+      drwxr-xr-x  32 tw tw     4096 Feb 20 10:19 include
+      drwxr-xr-x   2 tw tw     4096 Feb 25 16:25 init
+      drwxr-xr-x   2 tw tw     4096 Feb 21 11:01 ipc
+      -rw-r--r--   1 tw tw     2245 Feb 20 10:17 Kbuild
+      -rw-r--r--   1 tw tw      563 Feb 20 10:17 Kconfig
+      drwxr-xr-x  20 tw tw    12288 Feb 25 16:17 kernel
+      drwxr-xr-x  13 tw tw    20480 Feb 21 11:01 lib
+      drwxr-xr-x   5 tw tw     4096 Feb 20 10:17 LICENSES
+      -rw-r--r--   1 tw tw 17572590 Feb 25 16:17 linux-4.19.59-xenomai-3.1-tw+-x86.tar.gz
+      ...
+
+Launch RTVM
+***********
+
+#. Prepare a dedicated disk (NVMe or SATA) for RTVM, just as you did :ref:`for Preempt-RT <install_rtvm>`, here we suppose it's ``/dev/sda1``.
```

Comment:
Is it correct that the only steps one should follow are to 1) download the `preempt-rt` image and 2) flash it to that disk partition? If true, I'd recommend we write them down here. It would duplicate the info a tiny bit but we could avoid the risk of the image version getting out of sync and we could also avoid that people get confused as to how far they should follow the instructions in the `install_rtvm` section of that other document.

Reply:
Yes, make sense.